tensorflow/python/ops/image_ops_impl.py

```diff
@@ -1501,10 +1501,12 @@ def decode_image(contents, channels=None, name=None):
     contents: 0-D `string`. The encoded image bytes.
     channels: An optional `int`. Defaults to `0`. Number of color channels for
       the decoded image.
+    dtype: The desired `tf.DType` of the returned `Tensor`. Must be either
+      `tf.uint8` or `tf.uint16`.
```

Comment:
Update the comment (we no longer restrict to `tf.uint8` and `tf.uint16`. So just:

```python
dtype: The desired DType of the returned `Tensor`.
```